README.md

```diff
@@ -1,5 +1,40 @@
 [![Automatic version updates](https://github.com/ZOSOpenTools/gnport/actions/workflows/bump.yml/badge.svg)](https://github.com/ZOSOpenTools/gnport/actions/workflows/bump.yml)
 
-gn
+# gnport
 
 A port of the Google Generate Ninja (gn) meta-build system to z/OS Open Tools project
+
+
+### Developer Notes
+
+In case of an error, use this workflow to ensure the file system is in 
+pristine state.
+
+```
+$ zopen remove -p zoslib
+$ zopen-clean --all -c -d -u -v
+$ zopen clean -d
+```
+
+Afterwards, examine the file system for no stale binaries.
+
+```
+$ find . -name "libzoslib*"
+```
+
+A good build will have results in ~/zopen as shown:
```

Comment:
Suggest adding "(though timestamps shown may differ)" after 'as shown'.

Reply:
Yes, Gaby, that is technically correct, but can we get this approved?  Put this one to bed and work on depot_tools.

Reply:
Without the suggested text, a user may think that the specific 20240110_084622 build of zoslib must be used.

Reply:
Sounds good.  I added a note to reflect that timestamps will vary.

Reply:
Thanks for the approval Gaby. I appreciate your time and expertise!